src/test/chmod.c

```diff
@@ -13,6 +13,8 @@ int main(void) {
   test_assert(0 == access(file_path, W_OK));
   test_assert(0 == fchmodat(AT_FDCWD, file_path, 0400, 0));
   test_assert(0 == access(file_path, R_OK));
+  test_assert(0 == faccessat(AT_FDCWD, file_path, 0400, AT_SYMLINK_NOFOLLOW));
+  test_assert(0 == faccessat2(AT_FDCWD, file_path, 0400, AT_SYMLINK_NOFOLLOW));
```

Comment:
Like I said, this will return `ENOSYS` on many systems so please check for that and let the test pass if it happens.

Reply:
Right. Sorry about that. Does the latest version do it correctly?